custom/icds_reports/utils/aggregation_helpers/distributed/agg_bihar_api_demographics.py

```diff
@@ -0,0 +1,110 @@
+
+from custom.icds_reports.const import AGG_BIHAR_API_DEMOGRAPHICS, AGG_MIGRATION_TABLE
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import BaseICDSAggregationDistributedHelper
+from corehq.apps.userreports.util import get_table_name
+from dateutil.relativedelta import relativedelta
+from custom.icds_reports.utils.aggregation_helpers import transform_day_to_month, month_formatter
+
+
+class AggBiharApiDemographicsHelper(BaseICDSAggregationDistributedHelper):
```

Comment:
Given the size of the data, this is likely to be very slow, especially as the table gets bigger. I think this table should probably be partitioned by month to keep this reasonable, although I don't have a sense at all how fast it will be

Reply:
sure. I can make it partitioned.

Reply:
I have done partitioned with month, Please have a look